torch_harmonics/plotting.py

```diff
@@ -0,0 +1,187 @@
+# coding=utf-8
+
+# SPDX-FileCopyrightText: Copyright (c) 2022 The torch-harmonics Authors. All rights reserved.
```

Comment:
since this is a completely new file, do we want to add 2025 license header as well?